upload/library/Sedo/TinyQuattro/Helper/MiniParser.php

```diff
@@ -1050,7 +1050,7 @@ protected function _pushClosingTagSuccess($tagName)
 		/*Plain Text Mode: disable*/
 		if(!empty($tagRules['plainText']) && $tagRules['plainText'] == $tagName)
 		{
-			$this->_plainText = false;
+			$this->_plainTextMode = false;
```

Comment:
Damn, this isn't enough to make nested [plain][plain][/plain][/plain] work as expected